dynamic_programming/trapped_water.py

```diff
@@ -12,8 +12,7 @@
 on both sides minus height of bar at current position.
 """
 
-
-def trapped_rainwater(height: list[int]) -> int:
+def trapped_rainwater(height : list[int] = None) -> int:
```

Comment:
Let's not give a default value.  _Force_ the caller to give you data to work with.

Reply:
Okay !